README.md

```diff
@@ -2,12 +2,18 @@
 
 Repository for out-of-tree scheduler plugins based on scheduler framework.
 
-Official container images are coming soon. They will eventually be available at the below location.
+## Install
+
+Starting with release v0.18.9 container images are available in the official k8s container registry
+`k8s.gcr.io/scheduler-plugins/kube-scheduler`.
+
 ```shell
-docker pull k8s.gcr.io/scheduler-plugins/kube-scheduler:$TAG
-docker pull k8s.gcr.io/scheduler-plugins/controller:$TAG
+docker pull k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.18.9
```

Comment:
If there is a list showing the existing tags, it may be more convenient for users to choose.

Reply:
I could build a mark down table right here in the README similar to what we the descheduler does. This would need to be maintained by hand.

https://github.com/kubernetes-sigs/descheduler#compatibility-matrix

It doesn't look like the `docker` command supports listing images or tags from a remove registry. I think I could come up with a curl command to list the tag in the registry. I think using `gcloud` would be an option too, but most people probably won't have that installed.

Do you prefer a documented curl command or a table in the mark down file?

Reply:
I think a mark down table in README is better. it may be more convenient for users to choose the version.
We can modify it manually after release.
WDYT?

Reply:
Yep ... that is fine. I'll update the PR to add this change. Thanks for the feedback.

Reply:
I amended the commit and added a compatibility matrix to the README.md file. Let me know if you want any additional changes.